utils/storage.js

```diff
@@ -10,22 +10,34 @@
  */
 'use strict';
 
-export function get(key: any, defaultValue: any = null): any {
-  let value;
-  try {
-    value = JSON.parse((localStorage.getItem(key): any));
-  } catch (error) {
-    console.error('Could not read from localStorage.', error);
-  }
-  return value !== null ? value : defaultValue;
-}
+class WebStorage {
+  mechanism: Storage;
 
-export function set(key: any, value: any): boolean {
-  try {
-    localStorage.setItem(key, JSON.stringify(value));
-    return true;
-  } catch (error) {
-    console.error('Could not write to localStorage.', error);
+  constructor(mechanism: Storage) {
+    this.mechanism = mechanism;
   }
-  return false;
+
+  get = (key: any, defaultValue: any = null): any => {
+    let value;
+    try {
+      value = JSON.parse((this.mechanism.getItem(key): any));
+    } catch (error) {
+      console.error('Could not read from storage.', error);
+    }
+    return value !== null ? value : defaultValue;
+  };
+
+  set = (key: any, value: any): boolean => {
+    try {
+      this.mechanism.setItem(key, JSON.stringify(value));
+      return true;
+    } catch (error) {
+      console.error('Could not write to storage.', error);
+    }
+    return false;
+  };
+
 }
+
+export const local = new WebStorage(localStorage);
+export const session = new WebStorage(sessionStorage);
```

Comment:
Why session storage for this value vs local storage?

Reply:
@bvaughn to have separate state for this value on different pages/browser tabs

Reply:
Right, sorry. I understand that would be the result, but why is that the goal? I think that would be confusing if your DevTools settings panel had a value that was persisted differently for different pages. I think that would be a confusing UX.

Reply:
From my perspective it would be confusing to have one value shared between different applications for example, or different browser tabs. Also I believe there could be situations when the same application is opened in 2 different tabs and one has highlight updates enabled and other - disabled. In my mind that shouldn't be the same way as it's done for saving a theme in local storage which is shared between different instances.

But if that is not acceptable or indeed confusing - I can change implementation to use the local storage instead. Also if so - should I revert back changes regarding to introducing a wrapper for session storage and related refactoring as well?

Reply:
> But if that is not acceptable or indeed confusing - I can change implementation to use the local storage instead. Also if so - should I revert back changes regarding to introducing a wrapper for session storage and related refactoring as well?

Yeah, I think so– since I don't think we want to use session storage.

Reply:
@bvaughn OK, thanks.